src/bijectors/adbijector.jl

```diff
@@ -0,0 +1,53 @@
+"""
+Abstract type for a `Bijector{N}` making use of auto-differentation (AD) to
+implement `jacobian` and, by impliciation, `logabsdetjac`.
+"""
+abstract type ADBijector{AD, N} <: Bijector{N} end
+
+# AD implementations
+function jacobian(b::ADBijector{<:ForwardDiffAD}, x::Real)
```

Comment:
Can we merge the forward and inverse functions?

Reply:
You mean `Union{Inversed{B}, B}` where `B<:ADBijector{<:ForwardDiff}`? If so, that seems like a good idea to me:)

Reply:
Yes.